src/App/Fossa/API/BuildWait.hs

```diff
@@ -84,15 +85,16 @@ waitForIssues ::
   , Has (Lift IO) sig m
   ) =>
   ProjectRevision ->
+  Maybe DiffRevision ->
   Cancel ->
   m Issues
-waitForIssues revision cancelFlag = do
+waitForIssues revision maybeDiffRevision cancelFlag = do
```

Comment:
[nit, optional] Can we avoid the [Hungarian Notation](https://en.wikipedia.org/wiki/Hungarian_notation) here? We already know it's `Maybe` from the type, and aren't trying to avoid shadowing.
```suggestion
waitForIssues revision diffRevision cancelFlag = do
```

Only commenting here because this is the first time I'm seeing it, but I see lots of `maybe`-prefixed `diffRevisions` that I think would be more simply named without the prefix.